src/Core/Models/Api/Response/Accounts/PreloginResponseModel.cs

```diff
@@ -9,8 +9,12 @@ public PreloginResponseModel(UserKdfInformation kdfInformation)
     {
         Kdf = kdfInformation.Kdf;
         KdfIterations = kdfInformation.KdfIterations;
+        KdfMemory = kdfInformation.KdfMemory;
+        KdfParallelism = kdfInformation.KdfParallelism;
     }
 
     public KdfType Kdf { get; set; }
     public int KdfIterations { get; set; }
+    public int KdfMemory { get; set; }
+    public int KdfParallelism { get; set; }
```

Comment:
Should these be nullable and just default to null instead of 0?

Reply:
I made them nullable now wherever it is possible that the kdfType could be PBKDF2.